src/strife/s_sound.c

```diff
@@ -138,7 +138,7 @@ void S_Init(int sfxVolume, int musicVolume, int voiceVolume)
 {  
     int i;
 
-    I_SetOPLDriverVer(opl_v_new);
+    I_SetOPLDriverVer(opl_doom_1_9);
```

Comment:
Strife is based on Doom II v1.666, but I do *not* know for certain that it uses the same version of DMX as v1.666. This should be flagged for future investigation rather than simply assumed.